song_match/song/songs/mary_had_a_little_lamb.py

```diff
@@ -10,26 +10,34 @@
 from song_match.song import Song
 from song_match.song.note import Note
 
+EIGHTH_NOTE = .25 # time for eighth note
+QUARTER_NOTE = .5 # time for quarter note
+HALF_NOTE = 1 # time half note
+WHOLE_NOTE = 2 # time for whole note
 
 class MaryHadALittleLamb(Song):
 
     @property
     def _notes(self) -> List[Note]:
         return [
-            Note('C4'),
-            Note('D4'),
-            Note('E4')
+            Note('C4', HALF_NOTE),
```

Comment:
You wouldn't have to pass in `HALF_NOTE` here if the duration defaulted to some value.

Reply:
Done